docs/macros/doc_macros.py

```diff
@@ -0,0 +1,39 @@
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
```

Comment:
You're adding code to the docs folder, so ensure that our linters are processing this folder as well. They are defined in tox.ini. The ones you need are:
`isort, black, safety, bandit, gitleaks, black-check, isort-check,  flake8 , mypy , pylint , vulture , darglint`

Reply:
I was not sure where to place doc macros...  What is the suggested folder? scripts ?

Reply:
If they have to do with docs, I think it makes sense to have it where you placed it, but we need to update the linter commands to also look into the new directory.

Reply:
Moved to /scripts (simpler option). If @DavidMinarsch suggests other action, will fix it.